### DISCUSSION TITLE
Segmentation fault when artisan optimize

### DISCUSSION THREAD

**Discussion by soymgomez** [livewire/livewire#5999] [7. Bugs (archived)] [answered]
Hello,

we were testing Livewire v3 and encountered a problem.

When running `artisan optimize` in an project it generates a segmentation fault:

![image](https://github.com/livewire/livewire/assets/6207345/fecfedaa-5808-4630-8328-f98b943d229f)

We have tested this in a new project only with Laravel and Livewire and it still happens, to test it we have done the following:

```bash
curl -s "https://laravel.build/example-app" | bash
cd example-app
sail up -d
sail composer install
sail composer require livewire/livewire:^3.0@beta
sail artisan optimize
```

We don't see where the error can come from, any idea?

Best regards

**Comment by gehrisandro** [comment]
I ran into the similar issue. Here is a PR which resolved the issue for me: https://github.com/livewire/livewire/pull/6008

**Comment by joshhanley** [comment, marked as answer]
This has been fixed in PR https://github.com/livewire/livewire/pull/6008 and released in v3 beta.5.
